.gitignore

```diff
@@ -2,3 +2,4 @@ composer.lock
 composer.phar
 /vendor/
 
+/nbproject/
```

Comment:
Can you revert this line please? [Use a global `.gitignore` file instead](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).